x/ecocredit/server/msg_server.go

```diff
@@ -581,35 +581,31 @@ func (s serverImpl) Sell(goCtx context.Context, req *ecocredit.MsgSell) (*ecocre
 
 	for i, order := range req.Orders {
 
+		// verify expiration is in the future
+		if order.Expiration != nil && order.Expiration.Before(ctx.BlockTime()) {
```

Comment:
was there discussion on whether or not orders should be allowed to have no expiration? seems like a default expiry time might be nice to have (sorry to tack on yet another thing to this PR :-) )

Reply:
I don't recall a specific discussion but I do recall it being proposed as optional. Maybe that was just how I interpreted it. If nice to have and/or needs further discussion, I would prefer to see optional for now and a separate issue to revisit.

Reply:
created an issue here https://github.com/regen-network/regen-ledger/issues/697